core/src/epicli/data/common/ansible/playbooks/roles/repository/files/server/Debian/create-repository.sh

```diff
@@ -1,53 +1,57 @@
 #!/bin/bash -eu
 
-EPI_REPO_SERVER_PATH=$1 # /var/www/html/epirepo is the default
-IS_OFFLINE_MODE=$2
-script_path="$( cd "$(dirname "$0")" ; pwd -P )"
+epi_repo_server_path=$1 # /var/www/html/epirepo is the default
```

Comment:
That's fine, however I like convention when global variables are uppercase and local lowercase.

Reply:
I prefer the following:
exported variables / env variables = uppercase (like DEBIAN_FRONTEND=noninteractive )
global (reachable by all functions across one script) / local variables ( reachable only inside a function) = lowercase
lets talk about this on side :walking_man: